logistics/scripts/modules/settings/settings-management.psm1

```diff
@@ -290,7 +290,7 @@ function Add-TestHarnessSpecificAppSettings([hashtable] $Settings = @{ }, [strin
     $newSettings = @{
         ConnectionStrings = @{ }
         ApiSettings       = @{
-            Mode = "SharedInstance"
+            Mode = "sharedinstance"
```

Comment:
should we fix the api to not be case sensitive here?

Reply:
It's already not a case sensitive .

Reply:
can we revert the changes to this and the appsettings.json then?